README.md

```diff
@@ -18,6 +18,18 @@ advantages:
 * Better support for password managers (which often don't support basic
```

Comment:
The blurb for this needs updating too as this is now a pretty core feature, e.g. atm we have:

> Balrog is a lightweight authorization library for Ruby on Rails >= 5 written by Pixie Labs that can protect your routes with a single username & password combination.

And then a comparison to `http_basic_authentication_with`. It feels like we're lots more sophisticated than that now!